src/frontend/web_application/src/scenes/Settings/components/DevicesManagement/components/DisplayDevice/style.scss

```diff
@@ -0,0 +1,35 @@
+@import '../../../../../../styles/common';
+@import '../../../../../../styles/vendor/bootstrap_foundation-sites';
+
+.m-device {
+
+  &__row {
```

Comment:
I'm asking myself if it should be in FormRow directly with a modifier, something like `<FormRow gutter />` which add ... gutters